src/annotator/hypothesis-injector.js

```diff
@@ -74,3 +73,37 @@ export class HypothesisInjector {
     this._frameIdentifiers.delete(frame);
   }
 }
+
+/**
+ * Check if the iframe has already been injected
```

Comment:
```suggestion
 * Check if the Hypothesis client has already been injected into an iframe
```

Reply:
Apologies for that. I accepted the commit and GH automatically marked as resolved. However, I forgot to pull the commit to my local repo and it was lost.

I have fixed it.